src/sentry/deletions/defaults/group.py

```diff
@@ -21,24 +20,31 @@ def get_child_relations(self, instance):
             models.GroupRelease,
             models.GroupRedirect,
             models.GroupResolution,
-            models.GroupRuleStatus,
             models.GroupSnooze,
-            models.GroupTagValue,
             models.GroupTagKey,
-            models.GroupEmailThread,
             models.GroupSubscription,
             models.UserReport,
-            # Event is last as its the most time consuming
-            models.Event,
         )
+
+        # these are manually handled by the cleanup script
```

Comment:
I would also rather this be done inverse.

Instead of a flag `cleanup` as a boolean, what if we did `skip_models` as a list of models to remove from this list. That way cleanup can then just send this along since it knows what's going on.

Otherwise, I see these diverging and being harder to maintain.

Reply:
🤔 Ya I guess that makes complete sense.